requirement.txt

```diff
@@ -14,4 +14,39 @@ python-dotenv==1.0.0
 requests==2.31.0
 schedule==1.2.1
 urllib3==2.1.0
-yarl==1.9.4
\ No newline at end of file
+yarl==1.9.4
```

Comment:
it's seems like there're quite some redundant package, please clean up

Reply:
is there a way to check on this?